Source/Extensions/Blazorise.DataGrid/DataGrid.razor

```diff
@@ -5,17 +5,28 @@
 @using Blazorise.DataGrid.Models
 @* there are two CascadingValue because one is used only to define columns, and the other is to build rows based on those columns *@
 @* This one is to build rows *@
+
 <CascadingValue Value="@this" IsFixed>
     @if ( IsPagerVisible && ( PagerPosition == DataGridPagerPosition.Top || PagerPosition == DataGridPagerPosition.TopAndBottom ) )
     {
         @paginationFragment
     }
     <Form>
+        @if ( IsGroupableByColumn )
+        {
+            <_DataGridColumnDropZone TItem="TItem"

```

Comment:
The drag area must be opt-in. Some users would like not to show it and just have grouping enabled explicitly on columns.

Reply:
Agree. Tought about it but then forgot.
Should we do a specific parameter like GroupDrag or a more generic one like GroupEdit? Or GroupAssign?

Reply:
ShowGrouping on DataGrid?

Reply:
Kk